### PR TITLE
Integer codec

### PR DESCRIPTION
**Pull request by NorfairKing**

### CODE REVIEW

File: autodocodec-api-usage/test_resources/yaml-schema/integer.txt
Comment (CSVdB) [inline autodocodec-api-usage/test_resources/yaml-schema/integer.txt:1]: What is this about? This doesn't look entirely like text, at least using GitHub's fonts.

File: autodocodec-api-usage/test_resources/yaml-schema/natural.txt
Comment (CSVdB) [inline autodocodec-api-usage/test_resources/yaml-schema/natural.txt:1]: Dito

**Review by CSVdB — APPROVED** [review]
Looks good to me!

File: autodocodec-api-usage/test_resources/yaml-schema/integer.txt
Comment (NorfairKing) [reply autodocodec-api-usage/test_resources/yaml-schema/integer.txt:1]: It's github not supporting ANSI codes :)

**Pull request merged**